sopel_channelmgnt/channelmgnt/__init__.py

```diff
@@ -35,6 +42,9 @@ def configure(config):
     config.define_section('channelmgnt', ChannelmgntSection, validate=False)
     config.channelmgnt.configure_setting('datafile', 'Where is the datafile for channelmgnt?')
     config.channelmgnt.configure_setting('support_channel', 'What channel should users ask for help in?')
+    config.channelmgnt.configure_setting('global_chanops', 'If anybody, who should have chanop access in all channels? (if any then enter nickserv accounts)')
```

Comment:
We support global_chanops now, it's called inherits from

Reply:
Does having chanops and inherits from defined at the same time work?

Reply:
see my other PR

Reply:
Ok so I can't add that for us until that is merged